apps/xprof_gui/priv/src/components/monitoring/Graph/Graph.jsx

```diff
@@ -3,21 +3,55 @@ import React from 'react';
 import C3Chart from 'react-c3js';
 import 'c3/c3.css';
 import { AXIS, DATA, GRID, POINT, TRANSITION } from '../../../constants';
+import Grid from './Grid/Grid';
+
+function composeID(q) {
+  const modFuncArity = q.split(':');
```

Comment:
there used to be a utility function `chartId` which we stopped using, but it can be reintroduced for safe IDs https://github.com/Appliscale/xprof/blob/master/priv/app/utils.js#L2

Reply:
I'll use the `chartId(MFA)` function. If it worked before, it will work fine. Nevertheless I'll reintegrate it and test.